### PR TITLE
Page for Videos

### PR DESCRIPTION
**Pull request by Deep-Codes**
Based on #147

### CODE REVIEW

File: pages/videos/[id].js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import Breadcumb from '@components/Breadcumb';
+import VideoCard from '@components/VideoCard';
+import { videoData } from '@data/video';
+
+export const getStaticProps = async ({ params }) => {
+    console.log('***************************');
+    console.log(params);
+    console.log('***************************');
```
Comment (shrihari689) [inline pages/videos/[id].js:8]: Need this one? 🤔📍

File: pages/videos/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <section className="pt-10">
+            <div>
+                <Breadcumb list={[{ href: null, name: 'Videos' }]} />
+            </div>
+            <h2 class="text-3xl md:text-4xl xl:text-5xl tracking-tight mt-4 font-bold leading-tight dark:text-white">
+                Videos
+            </h2>
+            <div class="flex flex-col md:-mx-6 pt-2 text-gray-900 dark ">
+                {videoData.map((el) => (
+                    <VideoCard dt={el} />
```
Comment (shrihari689) [inline pages/videos/index.jsx:25]: Needed key? 🤔📌

**Review by shrihari689 — CHANGES_REQUESTED** [review]
I think Light Mode needed to be worked 📍

image

File: pages/videos/[id].js (hunk 1 above)
Comment (Deep-Codes) [reply pages/videos/[id].js:8]: Nope

File: pages/videos/index.jsx (hunk 2 above)
Comment (Deep-Codes) [reply pages/videos/index.jsx:25]: 🤦🏻‍♂️ Thanks Doing All Changes

**Review by shrihari689 — APPROVED** [review]
Looks good to me 🔥

**Pull request merged**